### PR TITLE
Small refactors

### PR DESCRIPTION
**Pull request by QGarchery**

### CODE REVIEW

File: test/helpers/SigUtils.sol [hunk 1]
```diff
@@ -1,8 +1,8 @@
-// SPDX-License-Identifier: GPL-2.0-or-later
+// SPDX-License-Identifier: MIT
```
Comment (QGarchery) [inline test/helpers/SigUtils.sol:1]: There were files with MIT, files with GPL-2.0-or-later, and files with UNLICENSED. I went for MIT

Comment (adhusson) [reply test/helpers/SigUtils.sol:1]: Don't know if it matters but in [metamorpho](https://github.com/morpho-org/metamorpho/blob/main/test/forge/helpers/SigUtils.sol) and [morpho-blue-bundlers](https://github.com/morpho-org/morpho-blue-bundlers/blob/main/test/forge/helpers/SigUtils.sol) this specific file for instance is GPL-2.0-or-later.

Comment (MathisGD) [reply test/helpers/SigUtils.sol:1]: yes we should go for GPL2

File: src/ERC20DelegatesUpgradeable.sol [hunk 2]
```diff
@@ -1,24 +1,23 @@
-// SPDX-License-Identifier: MIT
+// SPDX-License-Identifier: GPL-2.0-or-later
```
Comment (MerlinEgalite) [inline src/ERC20DelegatesUpgradeable.sol:1]: why?

File: src/MorphoToken.sol [hunk 3]
```diff
@@ -29,6 +29,12 @@ contract MorphoToken is ERC20DelegatesUpgradeable, ERC20PermitUpgradeable, Ownab
     /// @notice Reverts if the address is the zero address.
     error ZeroAddress();
 
+    /* CONSTRUCTOR */
+
+    constructor() {
```
Comment (MerlinEgalite) [inline src/MorphoToken.sol:34]: Missing natspecs

File: src/ERC20DelegatesUpgradeable.sol (hunk 2 above)
Comment (MathisGD) [reply src/ERC20DelegatesUpgradeable.sol:1]: it's the licence of all of our code, except the IRM (MIT) and Blue (BSL)

Comment (QGarchery) [reply src/ERC20DelegatesUpgradeable.sol:1]: See [conversation above ](https://github.com/morpho-org/morpho-token-upgradeable/pull/29#discussion_r1810055200), feel free to re-open that one above

**Review by MathisGD — APPROVED** [review]

**Pull request merged**